### PR TITLE
Update debug-visual-studio-code.md to specify app exe name

### PR DESCRIPTION
**Pull request by champnic**
Rendered article section for review:
* **Debug WebView2 apps with Visual Studio Code** > **Debug running processes**
   * https://review.learn.microsoft.com/microsoft-edge/webview2/how-to/debug-visual-studio-code?branch=pr-en-us-3240#debug-running-processes
   * [Before/live](https://learn.microsoft.com/microsoft-edge/webview2/how-to/debug-visual-studio-code#debug-running-processes)

Devs should specify a specific app exe name in the reg key when setting up debugging, rather than using '*' which will apply to all WebView2 apps.

AB#52889881

### CODE REVIEW

File: microsoft-edge/webview2/how-to/debug-visual-studio-code.md [hunk 1]
```diff
@@ -112,18 +112,18 @@ You might need to attach the debugger to running WebView2 processes.  To do that
...
 
    To do this, in the editor, navigate to `HKEY_CURRENT_USER\Software\Policies\Microsoft\Edge\WebView2\AdditionalBrowserArguments` by clicking on each subfolder under the path.
 
    If this path doesn't exist, navigate to `HKEY_CURRENT_USER\Software\Policies\Microsoft` in the editor, right-click the `Microsoft` folder, select **New**, and then select **Key**.  Enter `Edge` for the name of the new key.  Continue to do this for each subfolder until you have the full path: `HKEY_CURRENT_USER\Software\Policies\Microsoft\Edge\WebView2\AdditionalBrowserArguments`.
 
 1. Right-click the `AdditionalBrowserArguments` folder, select **New**, and then select **String Value**.
-Rename `New Value #1` to `*`.
+Rename `New Value #1` to `<yourApp.exe>`.
 
 1. Right click the **\*** value, and then select **Modify**.  Set the `Value Data` equal to `--remote-debugging-port=9222`.  Verify that the edit window matches the following:
```
Comment (victorhuangwq) [inline microsoft-edge/webview2/how-to/debug-visual-studio-code.md:128]:
```suggestion
1. Right click the **<yourApp.exe>** value, and then select **Modify**.  Set the `Value Data` equal to `--remote-debugging-port=9222`.  Verify that the edit window matches the following:
```

Also, I think we should update the screenshots in the following steps to match. I attached them here:

image
image

Comment (champnic) [reply microsoft-edge/webview2/how-to/debug-visual-studio-code.md]: I'm not sure how to properly update the images. Do either of you know and/or can add that change here?

Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/debug-visual-studio-code.md]: rendered article section:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/debug-visual-studio-code?branch=pr-en-us-3240#debug-running-processes

first image:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/debug-visual-studio-code-images/set-debugging-port.png

2nd image:
https://review.learn.microsoft.com/en-us/microsoft-edge/webview2/how-to/debug-visual-studio-code-images/set-debugging-port-registry-key.png

target local image dir:
C:\Users\v-mihoffman\GitHub\edge-developer\microsoft-edge\webview2\how-to\debug-visual-studio-code-images

quick & maybe lo-res update: I'll DL the above to that dir

Comment (mikehoffms) [reply microsoft-edge/webview2/how-to/debug-visual-studio-code.md]: pushed commit to rewrite reg editor ui steps & created fresh version of those two screenshot pngs
Pls tech review, @champnic ( / @victorhuangwq )

**Review by mikehoffms — APPROVED** [review]
Approved PR after pushing commits for Writer/Editor pass

**Review by champnic — COMMENTED** [review]
LGTM

**Review by victorhuangwq — APPROVED** [review]
LGTM

**Pull request merged**